docker/execute

```diff
@@ -0,0 +1,57 @@
+#!bin/bash
```

Comment:
Fix bash shebang syntax missing `/` after `#!` also use this version instead `#!/usr/bin/env bash`

Reply:
done